### PR TITLE
#131 PackageManager.getPackageInfo(packageName, 0) is deprecated as of…

### PR DESCRIPTION
**Pull request by co-dax**
… API 33: Android 13.0 (Tiramisu).

Close https://github.com/capawesome-team/capacitor-plugins/issues/131

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] The changes have been tested successfully.
- [ ] A changeset has been created (`npm run changeset`).
- [ ] I have read and followed the [pull request guidelines](https://capawesome.io/contributing/pull-requests/).

### CODE REVIEW

File: packages/app-update/android/src/main/java/io/capawesome/capacitorjs/plugins/appupdate/AppUpdatePlugin.java [hunk 1]
```diff
@@ -227,7 +229,13 @@ private boolean isGooglePlayServicesAvailable() {
 
     private PackageInfo getPackageInfo() throws PackageManager.NameNotFoundException {
         String packageName = this.getContext().getPackageName();
-        return this.getContext().getPackageManager().getPackageInfo(packageName, 0);
+
+        // PackageManager.getPackageInfo(packageName, 0) is deprecated as of API 33: Android 13.0 (Tiramisu).
```
Comment (robingenz) [inline packages/app-update/android/src/main/java/io/capawesome/capacitorjs/plugins/appupdate/AppUpdatePlugin.java:233]: Please remove the comment. The code should be enough.

File: packages/app-update/android/src/main/java/io/capawesome/capacitorjs/plugins/appupdate/AppUpdatePlugin.java [hunk 2]
```diff
@@ -227,7 +229,13 @@ private boolean isGooglePlayServicesAvailable() {
 
     private PackageInfo getPackageInfo() throws PackageManager.NameNotFoundException {
         String packageName = this.getContext().getPackageName();
-        return this.getContext().getPackageManager().getPackageInfo(packageName, 0);
+
```
Comment (robingenz) [inline packages/app-update/android/src/main/java/io/capawesome/capacitorjs/plugins/appupdate/AppUpdatePlugin.java:232]: Please remove the empty line.

**Review by robingenz — CHANGES_REQUESTED** [review]
Thank you for your contribution.

**Comment by co-dax** [comment]
Done.

**Pull request closed without merging**